src/main/java/org/isf/distype/dto/DiseaseTypeDTO.java

```diff
@@ -29,11 +29,11 @@
 public class DiseaseTypeDTO {
 
 	@NotNull
-	@Schema(description = "Disease type code")
+	@Schema(description = "Disease type code", maxLength = 2)
 	private String code;
 
 	@NotNull
-	@Schema(description = "Disease type description")
+	@Schema(description = "Disease type description", maxLength = 110)
```

Comment:
How did you come up with 110?  Seems like an odd number to me.
BTW, the next line is indented incorrectly.

Reply:
It is from the DDL: 

DCL_ID_A | char(2) PK
-- | --
DCL_DESC | varchar(110)
DCL_CREATED_BY | varchar(50)
DCL_CREATED_DATE | datetime
DCL_LAST_MODIFIED_BY | varchar(50)
DCL_LAST_MODIFIED_DATE | datetime
DCL_ACTIVE | tinyint(1)